lineman/app/services/inbox_service.coffee

```diff
@@ -0,0 +1,34 @@
+angular.module('loomioApp').factory 'InboxService', (UserAuthService, Records) ->
+  new class InboxService
+    constructor: ->
+      currentUser = UserAuthService.currentUser
+      baseThreadsView = (name) ->
+       view = Records.discussions.collection.addDynamicView(name)
```

Comment:
Oh, sorry!

I at one point for some reason thought this was trying to go in before the runthrough today.

Consider my jets cooled :snowflake: